tests/test_hn.py

```diff
@@ -95,13 +95,13 @@ def test_get_show_story_ids(self):
         self.assertEqual(len(show_story_ids), 10)
 
     def test_get_job_story_ids(self):
-        job_story_ids = self.client.get_job_story_ids(limit=10)
-        self.assertEqual(len(job_story_ids), 10)
+        job_story_ids = self.client.get_job_story_ids(limit=2)
+        self.assertEqual(len(job_story_ids), 2)
```

Comment:
Reducing assertion to 2 items as at the time of requesting, there were only 5 job stories available and this was causing this unit test to fail.